projects/storefrontlib/src/cms-components/checkout/cart/mini-cart/mini-cart.component.html

```diff
@@ -1,24 +1,9 @@
 <a
-  *ngIf="(cart$ | async) as cart"
-  aria-label="Cart"
-  [routerLink]="{ route: 'cart' } | cxTranslateUrl"
+  [attr.aria-label]="(quantity$ | async) + ' items currently in your cart'"
+  [routerLink]="{ route: ['cart'] } | cxTranslateUrl"
 >
-  <svg version="1.1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 35 28">
-    <g transform="translate(-4758 4746)">
-      <path
-        d="M4758.7-4746h4.7c0.3,0.1,0.6,0.3,0.7,0.5l1.7,7.5h23.6c0.4,0,0.7,0.4,0.7,0.8c0,0.1,0,0.1,0,0.2l-4,12
- c-0.1,0.2-0.4,0.4-0.7,0.4h-16.4l0.3,1.3h14.1c1.5,0,2.7,1.2,2.7,2.7c0,1.5-1.2,2.7-2.7,2.7l0,0c-1.5,0-2.6-1.2-2.6-2.6
- c0-0.5,0.1-1,0.4-1.4h-10.1c0.8,1.2,0.4,2.9-0.9,3.6c-0.4,0.3-0.9,0.4-1.4,0.4c-1.5,0-2.7-1.2-2.7-2.7c0-1.2,0.8-2.2,1.9-2.5
- l-5.1-21.4h-4.1c-0.3,0-0.6-0.2-0.7-0.6c0,0,0-0.1,0-0.1C4758-4745.7,4758.2-4746,4758.7-4746C4758.6-4746,4758.6-4746,4758.7-4746
- z"
-      />
-    </g>
-  </svg>
+  <cx-icon [type]="iconTypes.CART"></cx-icon>
```

Comment:
On mobile the styling has to be adjusted a bit
![Screen Shot 2019-04-25 at 4 34 55 PM](https://user-images.githubusercontent.com/10605724/56766745-2110db00-6778-11e9-80c4-9f807878f573.png)